src/docs/components/header/HeaderExamplesDoc.js

```diff
@@ -37,7 +37,7 @@ HeaderExamplesDoc.defaultProps = {
       props: { code: (
         <Header justify="between">
           <Title>Title</Title>
-          <Menu direction="row" align="center" responsive={false}>
+          <Menu direction="row" align="center" inline={false}>
```

Comment:
As indicated by the example label, this Header is supposed to have an inline menu.